app/assets/stylesheets/blacklight_gallery/_masonry.scss

```diff
@@ -0,0 +1,44 @@
+.view-type-masonry {
+  background-image: asset-url('icon-masonry.png') !important;
+  background-position: center;
+  background-repeat: no-repeat;
+  .view-icon-masonry {
+    padding-right: 12px;
+    height: 1.45em;
```

Comment:
I'm not sure if this one would be the same as above or not (again, weird thing with us faking our own icon).  I worry about if this would not look right if the variable was customized.